src/main/java/io/cdap/plugin/gcp/bigquery/sink/AbstractBigQuerySinkConfig.java

```diff
@@ -170,9 +106,5 @@ public void validate() {
                                                    "'.', '_', and '-'.", "bucket");
       }
     }
-
-    if (getWriteDisposition().equals(WriteDisposition.WRITE_TRUNCATE) && !getOperation().equals(Operation.INSERT)) {
```

Comment:
This validation was moved to class BigQuerySinkConfig
https://github.com/data-integrations/google-cloud/blob/2b73f3512fcaa692f94eea24ec05aed79378f102/src/main/java/io/cdap/plugin/gcp/bigquery/sink/BigQuerySinkConfig.java#L156-L162
because operation property available only for BigQuery Sink plugin, in BigQuery Multi Sink always using Operation.INSERT